joomla/joomla-cms
# Test for mcrypt support in the installer

Partial Pull Request for Issue #9507.
#### Summary of Changes

This adds a test to Joomla's install app to check for mcrypt support.  On a server without it, though Joomla is installable, a degraded experience will be present (namely two-factor authentication cannot be used and several libraries (JCrypt, FOFEncryptAes, the random_compat polyfill, and the password_compat polyfill) either use different encryption libraries or fail completely).
#### Testing Instructions

On the overview page of the install app, in the Pre-Installation Check section, you will see a new line testing for mcrypt support.  This should report accurately depending on your server's configuration.
